src/coreclr/jit/importer.cpp

```diff
@@ -14076,6 +14166,19 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 }
 
                 op1 = impPopStack().val;
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

Reply:
had removed.
Thanks